client/accept-invite/logged-out-invite/signup-form.jsx

```diff
@@ -103,6 +98,15 @@ export default React.createClass( {
 		)
 	},
 
+	footerLink() {
+		let logInUrl = config( 'login_url' );
+		if ( config.isEnabled( 'login' ) ) {
+			logInUrl = '/log-in';
```

Comment:
This doesn't work for me on my a8c account with 2fa enabled. I think we should remove this conditional, including lines `103-105`.